package.json

```diff
@@ -51,18 +54,32 @@
     "react-test-renderer": "18.0.0",
     "semantic-release": "^19.0.3"
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm run pretty-quick"
-    }
-  },
   "prettier": {
     "printWidth": 100,
     "singleQuote": true,
     "trailingComma": "all",
     "tabWidth": 2,
     "proseWrap": "always"
   },
+  "eslintConfig": {
+    "extends": "@callstack",
```

Comment:
We're not running Prettier anymore now, since we're not using any rule from `eslint-plugin-prettier`

Reply:
Prettier check is triggered by `@callstack` ESLint config (I've checked that it throws errors). I've chosen that config, as it's the same we use for RNTL.

Also prettier is normally triggered by IDE (e.g. VSCode) format on save.

Reply:
According to the docs, `eslint-plugin-prettier` isn't run with the default config

https://github.com/callstack/eslint-config-callstack#react-native-config